hand-eye-calibration/full_hand_eye_sample.py

```diff
@@ -13,69 +13,33 @@
 Further explanation of this sample is found in our knowledge base:
 https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/103481384/
 """
+import argparse
 from pathlib import Path
 import time
 import datetime
 
 import cv2
 import numpy as np
+from scipy.spatial.transform import Rotation as R
 import zivid
 import rtde.rtde as rtde
```

Comment:
Just curious, if rtde is used -> should we as requirement (in docs) state that UR robot is required?

Reply:
Maybe, but I think you can run the script with a UR simulator a well